src/components/SignUp.js

```diff
@@ -0,0 +1,97 @@
+import * as React from "react";
+import Button from "@mui/material/Button";
+import CssBaseline from "@mui/material/CssBaseline";
+import TextField from "@mui/material/TextField";
+import Box from "@mui/material/Box";
+import Typography from "@mui/material/Typography";
+import Container from "@mui/material/Container";
+
+export default function SignIn() {
+  const handleSubmit = (event) => {
+    event.preventDefault();
+    const data = new FormData(event.currentTarget);
+    console.log({
+      name: data.get("name"),
+      email: data.get("email"),
+      password: data.get("password"),
+    });
+  };
+
+  return (
+    <Container component="main" maxWidth="xs">
+      <CssBaseline />
+      <Box
+        sx={{
+          marginTop: 8,
+          display: "flex",
+          flexDirection: "column",
+        }}
+      >
+        <Typography component="h1" variant="h3">
+          Welcome!
+        </Typography>
+        <Box component="form" onSubmit={handleSubmit} noValidate sx={{ mt: 1 }}>
+          <TextField
+            margin="normal"
+            required
+            fullWidth
+            id="name"
+            label="Full Name"
+            name="name"
+            autoComplete="name"
+            autoFocus
+          />
+          <TextField
+            margin="normal"
+            required
+            fullWidth
+            id="email"
+            label="Email Address"
+            name="email"
+            autoComplete="email"
+          />
+          <TextField
+            margin="normal"
+            required
+            fullWidth
+            name="password"
+            label="Password"
+            type="password"
+            id="password"
+            autoComplete="current-password"
```

Comment:
We have more input fields on a Sign up page. Please refer to work flowchart and wireframes. Thanks

Reply:
I was going by the design in this miro page. https://miro.com/app/board/uXjVPM3mqMI=/
According to this page, we have three fields in the sign up page, "Full Name", "Email Address" and "Password". And then there are two buttons. 
![Screen Shot 2022-10-30 at 8 17 15 AM](https://user-images.githubusercontent.com/88150284/198886522-743fdbfa-6336-4c50-bd51-f970a9e85777.png)
Is there something I'm missing?

Reply:
Sign Up page should have a “Continue with GOOGLE” button, as we discussed in the last meeting. 
“Already have an account? Login”
Because these two pages were assigned to you, could you please fix the code and wireframe also? Thank you

Reply:
Yes, Nuriye, probably you don't remember but you had first do wireframes according approve workflow chart. The wireframes you are talking about are kind of templates. They were not approved by everyone as a final wireframe. Thanks

Reply:
@XeniyaDob for the "already have an account?" we already had a sign-in button, the sign-in button will redirect the user to the sign-in page. I think that looks better and we already had it in the wireframe.
@elenamagay  I added the fourth input to confirm the password
Can you both review the latest update and approve it?